src/config/global.styles.ts

```diff
@@ -1,11 +1,11 @@
-import { createGlobalStyle } from 'styled-components';
-import './fonts/styles.css';
+// import { createGlobalStyle } from 'styled-components';
```

Comment:
And maybe we could remove this if we are not using it. But if we will use it in the future, I think we could leave it